tech/languages/python/python-installation.md

```diff
@@ -68,6 +70,12 @@ When you finish your work, just deactivate the virtual environment.
 
 ### What next?
 
- * [Python homepage](https://www.python.org/)
- * [Python 3 Documentation](https://docs.python.org/3/)
- * [Python Documentation: venv](https://docs.python.org/3/library/venv.html#module-venv)
+* [Python homepage](https://www.python.org/)
+* [Python 3 Documentation](https://docs.python.org/3/)
+* [Python Documentation: venv](https://docs.python.org/3/library/venv.html#module-venv)
```

Comment:
Sorry, I missed the question earlier. This change was just a cleanup, since lists in markdown are not indented.